test/MOIWrapper.jl

```diff
@@ -1,31 +1,38 @@
-using Base.Test, MathOptInterface, MathOptInterface.Test
+using MathOptInterface, MathOptInterface.Test
 
 const MOI  = MathOptInterface
+const MOIB = MathOptInterface.Bridges
 const MOIT = MathOptInterface.Test
 
 @testset "Linear tests" begin
     linconfig = MOIT.TestConfig(modify_lhs = false)
-    @testset "Default Solver"  begin
-        solver = ClpOptimizer(LogLevel = 0)
+    solver = ClpOptimizer(LogLevel = 0)
+    MOIT.contlineartest(solver, linconfig, [
         # linear1 test is disabled due to the following bug.
         # https://projects.coin-or.org/Clp/ticket/84
-        MOIT.contlineartest(solver, linconfig, ["linear1", "linear10","linear11",
-            "linear12","linear8a","linear8b","linear8c"])
+        "linear1",
+        "linear10", # linear10 test is tested below because it has interval sets
+        "linear12"  # incorrect certificate? Test 2 * cd1 ≈ -bndxd fails
```

Comment:
@blegat, you had experience with Clp returning incorrect farkas duals yeah? Because [this](https://github.com/JuliaOpt/MathOptInterface.jl/blob/ef6ec2381a34547e8f920a50700aa40ba900e7a4/src/Test/contlinear.jl#L1477) is the only test it fails. I get the solution `(cd1, cd2, bndxd, bndyd)=(-1/3, -1, 0,0)`.

Reply:
Yes, too much experience with Clp not being able to give the Farkas certificate. Here it gives it but it is incorrect, `bndxd` should be `2/3`.
I am not sure have already encountered Clp returning a certificate but a bad one, are we sure it is not a bug in getting duals of variable bounds in the wrapper ?

Reply:
How does one get the Farkas dual for a variable bound?

Reply:
Hmm so diving into the Cpp, there is an option to get the `fullray`:
https://github.com/coin-or/Clp/blob/e4cdb00a75f0ddc3b1e1b1f8b70919a963afd7f9/Clp/src/ClpModel.cpp#L2792-L2807
But that isn't wrapped at the C interface :(